app/views/users/sessions/new.html.slim

```diff
@@ -0,0 +1,16 @@
+= provide :title, 'Sign In'
+section.main-content.container
```

Comment:
this sort of thing could be added to a layout file of some sort. You know layouts can have layouts, etc...